core/cloudflow-akka/src/main/resources/reference.conf

```diff
@@ -3,3 +3,6 @@ akka.kafka.consumer.wakeup-debug = false
 
 # Tuning parameter of how many sends that can run in parallel.
 akka.kafka.producer.parallelism = 10000
+
+# Delay commits until a next offset is observed for batch operators like mapConcat
+akka.kafka.committer.when = NextOffsetObserved
```

Comment:
Didn't you have logic in Cloudflow to get the same effect?

Reply:
No that was removed quite some time ago.